index.html

```diff
@@ -38,12 +37,191 @@ <h1 class="greetings">
+          </li>
+          <li>
+            <a href="#">
+            <img
+              class="social linkedin"
+              src="image/icons/linkedin.png"
+              alt="linkedin icon"
+              />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+            <img class="social m" src="image/icons/m.png" alt="m icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+            <img
+              class="social github"
+              src="image/icons/github.png"
+              alt="github icon"
+              />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+            <img
+              class="social fin"
+              src="image/icons/fin.png"
+              alt="fin icon"
+              />
+            </a>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="works">
+      <div class="grid works-grid-container wrap padding-top-0">
+        <div class="flex works-grid-item">
+          <img
+            class="width-100"
+            src="image/tonic.jpg"
+            alt="portfolio screenshot"
+          />
+          <h3 class="section-title margin-top-12">portfolio</h3>
+          <div class="work-period">
+            <ul>
+              <li class="text-color-primary">Canopy</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p class="work-description text-color-primary margin-top-20">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="work-categories margin-top-12 padding-0">
+            <li><span>html</span></li>
+            <li><span>css</span></li>
+            <li><span>javascript</span></li>
+          </ul>
+          <div>
+          <button type="button" class="button button-enabled">
+            See Project
+          </button>
+          </div>
+        </div>
+        <div class="flex works-grid-item">
+          <img
+            class="width-100"
+            src="image/to-do-list.jpg"
+            alt="todolist screenshot"
+          />
+          <h3 class="section-title margin-top-12">to do list app</h3>
+          <div class="work-period">
+            <ul>
+              <li class="text-color-primary">Canopy</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p class="work-description text-color-primary margin-top-20">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="work-categories margin-top-12 padding-0">
+            <li><span>html</span></li>
+            <li><span>css</span></li>
+            <li><span>javascript</span></li>
+          </ul>
+          <div>
+          <button type="button" class="button button-enabled">
+            See Project
```

Comment:
- [ ] kindly align this section as well following the same figma design, margins and padding should be respected
your design 👇🏾
![Screenshot (62)_LI](https://user-images.githubusercontent.com/65298048/170495044-6ca7d5d6-bb56-4b1f-917c-8e87350e94fd.jpg)
figma design 👇🏾 
![Screenshot (61)_LI](https://user-images.githubusercontent.com/65298048/170495102-7194ce30-73ad-4598-8297-152168c48393.jpg)